docs/quickstart.rst

```diff
@@ -96,6 +96,7 @@ Go to the `SR Linux` folder and then `containerlab` to spin-up the containers:
 
     cd examples/Networking/SR\ Linux/containerlab
     sudo docker pull ghcr.io/nokia/srlinux:latest
+    sudo docker pull ghcr.io/inmanta/orchestrator:latest
```

Comment:
I don't know about this one. I'm not sufficiently familiar with the containers to review this part.

Reply:
@bartv wdyt ? 
For reference this is the corresponding topology file https://github.com/inmanta/examples/blob/master/Networking/SR%20Linux/containerlab/topology.yml

Reply:
The image looks good, but I am not sure why we are pulling it here, containerlab will do it too

Reply:
If containerlab can find an image with the same name locally it will use it instead of actually pulling the latest one.
I think I added this line here because I ran into this issue